test/Rakefile

```diff
@@ -64,12 +64,13 @@ namespace :compose do
   task :teardown_master do
     sh "docker-compose stop api mongodb"
     sh "docker-compose rm --force api mongodb"
+    sh "kontena master remove --force compose-e2e"
   end
 
   task :teardown_grid do
     sh "docker-compose stop agent"
     sh "docker-compose rm --force agent"
-    sh "kontena node rm --force moby"
+    sh "kontena node rm --force $HOSTNAME"
```

Comment:
Duh, can't `docker-compose run agent hostname` because it tries to do some kind of linking for the `container_name: kontena-agent` part, which fails with `network_mode: host`...

```
compose.cli.verbose_proxy.proxy_callable: docker create_host_config -> {'Binds': [u'/var/run/docker.sock:/var/run/docker.sock:rw'],
 'Links': ['kontena-agent:agent', 'kontena-agent:kontena-agent'],
 'LogConfig': {'Config': {}, 'Type': u''},
 'NetworkMode': 'host',
 'PortBindings': {},
 'VolumesFrom': []}
compose.cli.verbose_proxy.proxy_callable: docker create_container <- (tty=True, stdin_open=True, name=u'test_agent_run_1', image=u'test_agent', labels={u'com.docker.compose.version': u'1.11.1', u'com.docker.compose.container-number': '1', u'com.docker.compose.service': u'agent', u'com.docker.compose.project': u'test', u'com.docker.compose.oneoff': u'True'}, host_config={'NetworkMode': 'host', 'Links': ['kontena-agent:agent', 'kontena-agent:kontena-agent'], u'Isolation': None, 'PortBindings': {}, 'Binds': [u'/var/run/docker.sock:/var/run/docker.sock:rw'], 'LogConfig': {'Type': u'', 'Config': {}}, 'VolumesFrom': []}, environment=[u'KONTENA_URI=ws://localhost:9292', u'KONTENA_TOKEN=e2etoken'], command=['hostname'], volumes={u'/var/run/docker.sock': {}}, detach=False, ports=[])
ERROR: compose.cli.main.main: Cannot create container for service agent: Conflicting options: host type networking can't be used with links. This would result in undefined behavior